todo_app/templates/index.html

```diff
@@ -2,34 +2,36 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
+  <div class="block-header">
+    <h1 class="block-header--title">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
   <div class="row">
-    <div class="col-md-12">
+    <div class="task-list">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        {% for item in items %}
-        <li class="list-item">
-          <form action="/update" method="post">
-            <label for="complete">{{ item.title }}</label>
-            <input type='hidden' id="complete" name="item-id" value = {{ item.id }}>
-            <input type="checkbox" onChange='submit()' {% if item.status == "Complete" %} checked {% endif %}>
-          </form>
-        </li>
+        {% for card in cards %}          
+            <li class="list-item">  {# add status in #}
```

Comment:
I _think_ this todo is complete!
```suggestion
            <li class="list-item"> 
```